rules/windows/file_event/sysmon_susp_procexplorer_driver_created_in_tmp_folder.yml

```diff
@@ -15,13 +15,13 @@ logsource:
     category: file_event
 detection:
     selection_1:
-        TargetFilename: '*\AppData\Local\Temp\\*\PROCEXP152.sys'
+        TargetFilename|endswith: '\AppData\Local\Temp\\*\PROCEXP152.sys'
     selection_2:
         Image|contains:
```

Comment:
should be `|endswith` here

Reply:
Sorry if misunderstood, but it already endswith

Reply:
Well, not I am also misunderstood this :D As far as I remember I left this comment next to the `startswith` modifier, but now I cannot find it. My browser is extremely buggy right now, it is a huge PR